test_scripts/API/Navigation/UnsubscribeWayPoints/006_UnsubscribeWayPoints_GENERIC_ERROR_HMI_did_not_respond.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/27
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Unsubscribe_from_Destination_and_Waypoints.md
+-- Item: Use Case 1: Exception 4
+--
+-- Requirement summary:
+-- [UnsubscribeWayPoints] As a mobile app I want to be able to unsubscribes from getting notifications
+-- about any changes to the destination or waypoints.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sent valid UnsubscribeWayPoints_request to SDL
+-- 2) and HMI did not respond during default timeout
+--
+-- SDL must:
+-- 1) respond GENERIC_ERROR, success:false to mobile application
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonNavigation = require('test_scripts/API/Navigation/commonNavigation')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+
+--[[ Local Functions ]]
+local function unsubscribeWayPoints(self)
+  local cid = self.mobileSession1:SendRPC("UnsubscribeWayPoints", {})
+  EXPECT_HMICALL("Navigation.UnsubscribeWayPoints")
+  :Do(function()
+      -- HMI does not respond
+    end)
+  self.mobileSession1:ExpectResponse(cid, { success = false, resultCode = "GENERIC_ERROR" })
+  commonTestCases:DelayedExp(11000)
```

Comment:
@dboltovskyi propose to replace with
`commonTestCases:DelayedExp(commonNavigation.timeout + 1000)`

Reply:
@AKalinich-Luxoft Disagree
In this way total timeout will be 3s, but case requires 11s timeout